avm/res/service-bus/namespace/main.bicep

```
@@ -455,6 +458,52 @@ resource serviceBusNamespace_roleAssignments 'Microsoft.Authorization/roleAssign
   }
 ]
 
+module secretsExport 'modules/keyVaultExport.bicep' = if (secretsExportConfiguration != null) {
+  name: '${uniqueString(deployment().name, location)}-secrets-kv'
+  scope: resourceGroup(
+    split((secretsExportConfiguration.?keyVaultResourceId ?? '//'), '/')[2],
+    split((secretsExportConfiguration.?keyVaultResourceId ?? '////'), '/')[4]
+  )
+  params: {
+    keyVaultName: last(split(secretsExportConfiguration.?keyVaultResourceId ?? '//', '/'))
+    secretsToSet: union(
+      [],
+      contains(secretsExportConfiguration!, 'rootPrimaryConnectionString')
+        ? [
+            {
+              name: secretsExportConfiguration!.rootPrimaryConnectionString
```

```suggestion
              name: secretsExportConfiguration!.?rootPrimaryConnectionString
```
... to address a new Bicep warning (even try get technically should not be needed here)...